src/lib/image/image.shim.js

```diff
@@ -37,7 +38,13 @@ var ImageUtil = {
         metadata.image = dimensions;
 
         var ctx = canvas.getContext('2d');
-        ctx.drawImage(img, 0, 0, dimensions.width, dimensions.height);
+        if (file && file.image && file.image.orientation && file.image.orientation !== 1) {
+          ImageOrientationUtil.setImageOrientation({orientation: file.image.orientation, img, width: dimensions.width, height: dimensions.height, ctx});
```

Comment:
Same comment from above: do both branches of this if block belong in setImageOrientation?